cmd/thanos/query.go

```diff
@@ -155,6 +158,17 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Errorf("Address %s is duplicated for --metadata flag.", dup)
 		}
 
+		// Check if the YAML configuration of request.logging is correct.
+		httpLogOpts, err := logging.DecideHTTPFlag(*reqLogDecision, reqLogConfig)
+		if err != nil {
+			return errors.Wrapf(err, "error while parsing config for request logging")
+		}
+
+		tagOpts, grpcLogOpts, err := logging.DecideGRPCFlag(*reqLogDecision, reqLogConfig)
```

Comment:
Open question: Do we need separate `tagOpts`?

Reply:
Would be nice to keep the options separate, combining them would create unnecessary complexity :)